test/compile/data/timeunit.test.ts

```diff
@@ -55,7 +55,9 @@ describe('compile/data/timeunit', () => {
         }
       });
       const timeUnitNode = TimeUnitNode.makeFromEncoding(null, model);
-      expect(timeUnitNode.hash()).toBe('TimeUnit {"month_a":{"as":"month_a","field":"a","timeUnit":"month"}}');
+      expect(timeUnitNode.hash()).toBe(
+        'TimeUnit {"{\\"as\\":\\"month_a\\",\\"field\\":\\"a\\",\\"timeUnit\\":\\"month\\"}":{"as":"month_a","field":"a","timeUnit":"month"}}'
```

Comment:
Use ` to avoid escaping.

Reply:
I think it remains the same with string literals since backslash still needs escaping

Reply:
Good point. I think we should switch to using ' for escaping strings in all of Vega-Lite but that's a separate issue.